no_of_word.rb

```diff
@@ -0,0 +1,10 @@
+str=gets.chomp
+user = Hash.new
+for i in(0...str.length)
+    if user[str[i]].nil?
+        user[str[i]] = 1
+    else
+        user[str[i]] += 1
+    end
+end
```

Comment:
1. Use 2 space indentation
2. Use proper variable names
3. Is there any in-built method in ruby to find the frequency